modules/ossm-security-cert-manage.adoc

```diff
@@ -0,0 +1,130 @@
+// Module included in the following assemblies:
+//
+// * service_mesh/service_mesh_user_guide/ossm-security.adoc
+
+[id="ossm-cert-manage"]
+= Adding an external certificate authority key and certificate
+
+By default, {ProductName} generates self-signed root certificate and key, and uses them to sign the workload certificates. You can also use the operator-specified certificate and key to sign workload certificates, with operator-specified root certificate. This task demonstrates an example to plug certificates and key into {ProductShortName}.
+
+.Prerequisites
+
+* You must have installed {ProductName} with mutual TLS enabled to configure certificates.
+* You must use the certificates from link:https://github.com/maistra/istio/tree/maistra-2.0/samples/certs[Maistra repo]. 
```

Comment:
This makes it sound like you must use the Maistra certificates for production use, when I think you're just saying you need them for this example.  Maybe rephrase something like "The following example requires that you use the certificates from the Maistra repo.  In production you would use [other source - their own certificates?].

Reply:
done